SkietbaanBE/SkietbaanBE/Controllers/FeaturesController.cs

```diff
@@ -32,5 +32,23 @@ public User Search(string username)
             }
             return null;
         }
+
+        //// PUT: api/User/Update
+        [HttpPut]
+        [ActionName("Update")]
+        public async Task<IActionResult> PutUserMember(String username, [FromBody] User user)
+        {
+            User dbUser = null; //assume user does not exist
+            dbUser = _context.Users.Where(u => u.Username == user.Username)
```

Comment:
The FirstOrDefault() will set dbUser to null when not found, no need to have `User dbUser = null;`.

Make dbUser a `var`

Reply:
Have made the change